NOnion/NOnion.fsproj

```diff
@@ -97,7 +98,7 @@
 
   <Target Name="CopyProjectReferencesToPackage" DependsOnTargets="ResolveReferences">
     <ItemGroup>
-      <BuildOutputInPackage Include="@(ReferenceCopyLocalPaths->WithMetadataValue('ReferenceSourceTarget', 'ProjectReference'))" />
+      <BuildOutputInPackage Include="@(ReferenceCopyLocalPaths-&gt;WithMetadataValue('ReferenceSourceTarget', 'ProjectReference'))" />
```

Comment:
this change is unrelated to this commit, please separate it to a new commit

Reply:
@aarani if you dont include this change in a 3rd commit, it will keep showing up.

Reply:
Done.